app/actions/device/index.js

```diff
@@ -1,14 +1,19 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import {networkStatusChanged} from 'redux-offline/lib/actions';
+import {Client4} from 'mattermost-redux/client';
+
 import {DeviceTypes} from 'app/constants';
 
 export function connection(isOnline) {
-    return async (dispatch, getState) => {
+    return async (dispatch) => {
+        Client4.setOnline(isOnline);
+        dispatch(networkStatusChanged(isOnline));
```

Comment:
Have not seen the internal yet but is it the same with `networkStatusChangedAction` which we're using at `mattermost-redux`?

Reply:
actually is the same one, I'll change it to use that one